README.md

```diff
@@ -112,6 +112,18 @@ The following properties are accepted:
 
   When the value is `esbuild_zisi`, `esbuild` will be used with a fallback to `zisi` in case of an error.
 
+- `nodeVersion`
+
+  _Type_: `string`\
+  _Default value_: `12.x`
+
+  The version of Node.js to use as the compilation target. Possible values:
+
+  - `8.x` (or `nodejs8.x`)
+  - `10.x` (or `nodejs10.x`)
+  - `12.x` (or `nodejs12.x`)
+  - `14.x` (or `nodejs14.x`)
```

Comment:
Would it be simpler to allow only the AWS values, i.e. `nodejs8.x` but not `8.x`?

Reply:
It's simpler, but from an API perspective it feels redundant to include `nodejs` in the value of a property that relates specifically to the Node.js version (e.g. `nodeVersion: "nodejs10.x"`), so I thought it'd be nice to allow the shorter syntax too.

It's not essential though, so I'm happy to revisit if you think this is a bad idea.

Reply:
:+1: